### ISSUE TITLE
What should the D-loss and G-loss look like theoretically?

### ISSUE THREAD

**Issue by knaffe** [affinelayer/pix2pix-tensorflow#61]
While I train my model, the D-loss and the G-loss will have different tendency with different setting of batch size.
In other words, the D-loss would increase while the G-loss would drop down. Sometimes they are contrary.
So, what should the two loss like in theory? How could I say that I get a good model?

**Comment by krwarner** [comment]
In theory the sum of D-loss and G-loss (mathematically equivalent to the negative of your D-loss) should equal 0. So in theory the closer the D-loss and L1 G-loss, the better your model. This is a good post to learn more about it https://medium.com/@ekss1121/generative-adversarial-networks-b9f80e6d7679
